docs_user/assemblies/assembly_configuring-isolated-networks.adoc

```diff
@@ -6,29 +6,18 @@ ifdef::context[:parent-context: {context}]
-* IP range, per isolated network, for the data plane nodes. (These ranges will
-  configure data plane nodes to `NetConfig` CRs.) For more information, see xref:configuring-data-plane-nodes_isolated-networks[Configuring data plane nodes].
-* IP range, per isolated network, for control plane services. (These ranges
-  will enable pod connectivity to isolated networks to
-  `NetworkAttachmentDefinition` CRs.) For more information, see xref:configuring-networking-for-control-plane-services_isolated-networks[Configuring the networking for control plane services].
-* IP range, per isolated network, for load balancer IP addresses. (These ranges will define load balancer IP addresses to `IPAddressPool` CRs for MetalLB.) For more information, see xref:configuring-networking-for-control-plane-services_isolated-networks[Configuring the networking for control plane services].
-
-[IMPORTANT]
-Make sure you have the information listed above before proceeding with the next steps.
+* 1 IP address, per isolated network, per {rhocp_long} worker node. These IP addresses configure {OpenShiftShort} worker nodes to the `NodeNetworkConfigurationPolicy` custom resources (CRs). For more information, see xref:configuring-openshift-worker-nodes_isolated-networks[Configuring {OpenShiftShort} worker nodes].
+* 1 IP range, per isolated network, for the data plane nodes. These ranges
+configure data plane nodes to `NetConfig` CRs. For more information, see xref:configuring-data-plane-nodes_isolated-networks[Configuring data plane nodes].
+* 1 IP range, per isolated network, for control plane services. These ranges
+enable pod connectivity to isolated networks to
+`NetworkAttachmentDefinition` CRs. For more information, see xref:configuring-networking-for-control-plane-services_isolated-networks[Configuring the networking for control plane services].
+* 1 IP range, per isolated network, for load balancer IP addresses. These IP ranges define load balancer IP addresses for MetalLB in the `IPAddressPool` CRs. For more information, see xref:configuring-networking-for-control-plane-services_isolated-networks[Configuring the networking for control plane services].
 
 [NOTE]
-The exact list and configuration of isolated networks in the examples
-listed below should reflect the actual adopted environment. The number of
-isolated networks may differ from the example below. IPAM scheme may differ.
+The exact list and configuration of isolated networks in this procedure should reflect the actual adopted environment. The number of isolated networks might differ from the examples used in this procedure. IPAM scheme might also differ.
 Only relevant parts of the configuration are shown. Examples are incomplete and
 should be incorporated into the general configuration for the new deployment,
 as described in the general {rhos_prev_long} documentation.
```

Comment:
A couple of questions on this note:

1. I’m a little bit confused by the sentence “Only relevant parts of the configuration are shown.” Does this mean the relevant parts to the adoption procedure?
2. In the last sentence in the note,  are we trying to say that the examples don’t necessarily reflect the customers’ configuration, and they need to use values that are specific to their configuration?
3. Does “adopted environment” refer to the RHOSO environment?